res/CircleSchema/0.8/circle_schema.fbs

```diff
@@ -623,6 +624,7 @@ union BuiltinOptions {
   BCQGatherOptions = 252,
   BCQFullyConnectedOptions = 253,
   InstanceNormOptions = 254,
+  RmsNormOptions = 255,
```

Comment:
Please use `250`, instead of `255`.

Reply:
Okay, I'll update it. 
ref) RightShiftOptions = 126, so we have 100+ rooms.